Source/ACE/StateMachines/Enum/MovementStates.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ACE.StateMachines.Enum
+{
+    public enum MovementStates : int
+    {
+        Idle,
+        Moving,
+        Arrived,
+        Abandoned,
+    }
+}
```

Comment:
done.   Sorry - I am aware of that now and will make sure I am doing it correctly going forward.